WordPress/openverse-frontend
# Add the component for the homepage grid

## Fixes




Related to #1433 by @panchovm.

## Description




This PR adds the gallery grid component shown on the redesigned home page. Care has been taken to minimise disruption to the existing homepage.

## Testing Instructions



0. Check out the branch and open Storybook.
1. Explore the story for `VHomeGallery`.
2. Compare the images with the mockups (including position).
3. Check their links and titles.
4. Resize width to see if the column count is updated.
5. Check animation with and without reduced motion.

---

0. Check out the branch and run the dev server.
1. Visit the home page with the `new_header` flag enabled (`http://localhost:8443/?ff_new_header=on`).
2. Observe the new home page.

## Checklist



      `Update index.md`).
      a parent feature branch.
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin



<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
